### PR TITLE
Submission of Lesson 2 assignment

### PR DESCRIPTION
**Pull request by peredayano**
Created DrawGrids.py file with functions required by Lesson 2 assignment

### CODE REVIEW

**Comment by peredayano** [comment]
Adding FizzBuzz exercise

**Comment by peredayano** [comment]
Still working on the series exercise, will complete today

**Comment by peredayano** [comment]
Completed assignments for Lesson 2 (DrawGrids, FizzBuzz, Series)

**Comment by peredayano** [comment]
Added testing to Series.py

**Comment by peredayano** [comment]
Please note that the Series.py (with tests function) in pull request is the most current version of the file. I have previously uploaded an incomplete Series.py file for the assignment but cannot re-upload the completed one. Please assess and grade the pull request Series.py file. Thanks.

File: students/ador_yano/Lesson02/DrawGrids.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        print("+")
+        for i in range(scale):                  # draw row per scale
+            for i in range(2):              # draw columns
+                print("|", "  " * scale, end="")
+            print("|")
+    for i in range(2):                      # draw bottom line
+        print("+", "- " * scale, end="")        # size side per scale
+    print("+")
+
+def print_grid2(a,b):
```
Comment (charlesrobison) [inline students/ador_yano/Lesson02/DrawGrids.py]: Looks good - might consider adding default values.

File: students/ador_yano/Lesson02/FizzBuzz.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+for x in range(1,101):
+    if (x % 3 == 0) & (x % 5 == 0):  # x is a multiple of 3 and 5
+        print("FizzBuzz")
+    elif x % 3 == 0:  # x is a multiple of 3
+        print("Fizz")
+    elif x % 5 == 0: # x is a multiple of 5
+        print("Buzz")
+    else:
+        print(x)
```
Comment (charlesrobison) [inline students/ador_yano/Lesson02/FizzBuzz.py]: Perfect!

File: students/ador_yano/Lesson02/Series.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+1. fib(n) - series starts with 0 and 1, returns nth value from zero index
+2. lucas(n) - series starts with 2 and 1, returns nth value from zero index
+3. sum_series(n, a, b) - optional arguments a and b default to 0 and 1, can be set to other values
+    returns nth value
+'''
+print(intro)
+
+def fib(n):
+    """ Return nth value in Fibonacci series """
+    if n < 0:                           # check for negative numbers
```
Comment (charlesrobison) [inline students/ador_yano/Lesson02/Series.py]: Nice use of catch for negative inputs!

File: students/ador_yano/Lesson02/DrawGrids.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# DrawGrids.py implements the Lesson 2 assignment from UWPCE Python Programming
```
Comment (natasha-aleksandrova) [inline students/ador_yano/Lesson02/DrawGrids.py]: note that python modules (python files) should follow snake_case convention -> `draw_grids.py`

File: students/ador_yano/Lesson02/Series.py (hunk 3 above)
Comment (peredayano) [reply students/ador_yano/Lesson02/Series.py]: Thanks for your feedback, Charles.

File: students/ador_yano/Lesson02/DrawGrids.py (hunk 4 above)
Comment (peredayano) [reply students/ador_yano/Lesson02/DrawGrids.py]: Thanks for the feedback, Natasha. I'll study the python style guidelines.

**Pull request merged**